meta-lmp-bsp/conf/machine/kv260.conf

```diff
@@ -38,3 +38,6 @@ SPL_FPGA_LOAD_ADDR = "0x18000000"
 
 # Make image-update essential as it is required for bootfw updates
 MACHINE_ESSENTIAL_EXTRA_RDEPENDS += "image-update"
+
+# SOM has infineon slb9670
+MACHINE_FEATURES += "tpm2"
```

Comment:
do you need a space here?

Reply:
no, we need the space when we use the `:append = " tpm2"`

Reply:
`+=` adds spaces itself. `:append` - doesn't. Yeah, a little unexpectedly :)